lib/src/widgets/chat/typing/ds_dot_animation.widget.dart

```diff
@@ -83,8 +86,8 @@ class _DSDotAnimationState extends State<DSDotAnimation>
     ).toList();
 
     for (int i = 0; i < widget.numberOfDots; i++) {
-      _animations.add(
-          Tween<double>(begin: 0, end: -20).animate(_animationControllers![i]));
+      _animations.add(Tween<double>(begin: 0, end: widget.upLevelAnimation)
+          .animate(_animationControllers![i]));
```

Comment:
my suggestion is to add the comma after the parentheses, so the code indentation will be better when saving the file

`      _animations.add(
        Tween<double>(begin: 0, end: widget.upLevelAnimation).animate(
          _animationControllers![i],
        ),
      );`